### PR TITLE
Adds documentation to deploy the UPF using the bundle

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

- Adds documentation to deploy the UPF using the sdcore-user-plane bundle instead of the standalone charm. 
- Adds documentation to integrate with COS.

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: docs/how-to/deploy_the_sdcore_user_plane.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
-# Deploy the SD-CORE UPF
+# Deploy the SD-CORE User Plane
 
-This guide explains how to deploy the SD-CORE UPF charmed operator as a standalone network 
-function on Kubernetes.
+This guide explains how to deploy the SD-CORE User Plane charm bundle.
 
 ## Requirements
 
 - Juju >= 3.0
```
Comment (sanchezfdezjavier) [inline docs/how-to/deploy_the_sdcore_user_plane.md:7]: We're using Juju `3.1.X` in the CI pipelines, should we require the same?

File: docs/how-to/integrate_the_sdcore_user_plane_with_cos.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Integrate the SD-CORE User Plane with COS
+
+This guide explains how to Integrate the SD-CORE User Plane with the Canonical Observability Stack (COS).
+
+## Requirements
+
+- Juju >= 3.0
```
Comment (sanchezfdezjavier) [inline docs/how-to/integrate_the_sdcore_user_plane_with_cos.md:7]: Same here for the Juju version.

File: docs/how-to/integrate_the_sdcore_user_plane_with_cos.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Integrate the SD-CORE User Plane with COS
```
Comment (sanchezfdezjavier) [inline docs/how-to/integrate_the_sdcore_user_plane_with_cos.md:1]: If people need to find this through an index, it's not obvious what COS is. I'd explicitly use the word Observability in the index and main title.

**Review by sanchezfdezjavier — APPROVED** [review]
Aside from the nitpicking LGTM

File: docs/how-to/deploy_the_sdcore_user_plane.md (hunk 1 above)
Comment (gruyaume) [reply docs/how-to/deploy_the_sdcore_user_plane.md]: Fixed

**Review by ghislainbourgeois — APPROVED** [review]

**Pull request merged**